### PR TITLE
WRP-12378: Fixed `Scroller` and `VirtualList` to focus Spottable container items properly via page up/down

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

When a page up/down key is pressed on items in a scroller or a list, an expected behavior is to scroll a page and focus an item at the same visual position normally.

And, if page up is pressed at the first page or page down is pressed at the last page, an expected behavior is to scroll to the edge then focus the first (for page up) or the last (for page down) item.

The reported issue is that, the focus disappears on page down key at the last page when the last item is spotlight container that has a spottable node not attached to the bottom of the item.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

`useScroll` has a logic to find an item to focus and it was working well _for items that are not Spotlight containers_ as it searches candidates from the last active Spotlight container. To fix this, I changed the starting node from the last active Spotlight container to a scroll container.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-12378

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Fixed
 
+- `sandstone/Scroller` and `sandstone/VirtualList` to handle focus properly via page up at the first page and via page down at the last page
```
Comment (MikyungKim) [inline CHANGELOG.md:13]:
```suggestion
- `sandstone/Scroller` and `sandstone/VirtualList` to handle focus properly via page up at the first page and page down at the last page
```

Comment (0x64) [reply CHANGELOG.md]: Done

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**